src/deepsparse/server/build_logger.py

```diff
@@ -93,9 +97,7 @@ def build_function_loggers(
             continue
         for target_logging_cfg in endpoint.data_logging:
             target = target_logging_cfg.target
-            target_identifier = _get_target_identifier(
-                endpoint.name or endpoint.task, target
-            )
+            target_identifier = _get_target_identifier(endpoint.name, target)
```

Comment:
I'm assuming this is safe now due to the validator added to `EndPointConfig`?

Reply:
yessir